### PR TITLE
SQL-1718: remove loglevel selector

### PR DESCRIPTION
**Pull request by terakilobyte**
This removes the loglevel selector from our connector. There are a lot of formatting changes too, I'm guessing power query updated their formatter. Better to knock it out now I suppose. The actual changes are minimal and I'll call them out in comments.

### CODE REVIEW

File: connector/MongoDBAtlasODBC.datasource-test.query.pq [hunk 1]
```diff
@@ -1,357 +1,326 @@
 section MongoDBAtlasODBCDataSourceTests;
-  shared IntegrationTests = [
-    uri = "mongodb://localhost/?ssl=false", 
-    DataSource = MongoDBAtlasODBC.Contents(uri, "supplies", "info", []),
+
+shared IntegrationTests = [
+    uri = "mongodb://localhost/?ssl=false",
+    DataSource = MongoDBAtlasODBC.Contents(uri, "supplies", []),
```
Comment (terakilobyte) [inline connector/MongoDBAtlasODBC.datasource-test.query.pq:5]: This is the only meaningful change here. Removing the "info" log level.

File: connector/MongoDBAtlasODBC.nativequery-test.query.pq [hunk 2]
```diff
@@ -3,11 +3,7 @@ section MongoDBAtlasODBCQueryTests;
 shared IntegrationTests = [
     uri = "mongodb://localhost/?ssl=false",
     Query_Complex_Types_Qualified = Value.NativeQuery(
-        MongoDBAtlasODBC.Contents(
-            uri, "integration_test", "info", []
-        ){
-            [Name = "integration_test", Kind = "database"]
-        }[Data],
+        MongoDBAtlasODBC.Contents(uri, "integration_test", []){[Name = "integration_test", Kind = "database"]}[Data],
```
Comment (terakilobyte) [inline connector/MongoDBAtlasODBC.nativequery-test.query.pq:6]: Removing "info" as we aren't sending a loglevel back.

File: connector/MongoDBAtlasODBC.pq [hunk 3]
```diff
@@ -26,23 +26,14 @@ MongoDBAtlasODBCType = type function (
...
-            Documentation.AllowedValues = {"ERROR", "INFO", "DEBUG"}
-        ]
-    ),
     optional options as record
 ) as table meta [
     Documentation.Name = "MongoDB Atlas SQL"
 ];
 
-MongoDBAtlasODBCImpl = (mongodbUri as text, database as text, optional loglevel as text, optional options as record) as table =>
+MongoDBAtlasODBCImpl = (mongodbUri as text, database as text, optional options as record) as table =>
```
Comment (terakilobyte) [inline connector/MongoDBAtlasODBC.pq:34]: Changes in this file are all meaningful, but thankfully small.

**Review by pmeredit — APPROVED** [review]
LGTM

**Review by nbagnard — APPROVED** [review]
LGTM!

And I suppose you tried it out manually too?

**Comment by terakilobyte** [comment]
I'll never push a change I haven't tried out manually :)

**Comment by terakilobyte** [comment]
evergreen merge

**Pull request merged**